README.md

```diff
@@ -48,6 +48,19 @@ Content:
    You should now have a working kubernetes cluster.
    You should also have an encrypted kubeconfig at `<CK8S_CONFIG_PATH>/.state/kube_config_<prefix>.yaml` that you can use to access the cluster.
 
+## Changing authorized SSH keys for a cluster
+
+Authorized SSH keys can be changed for a cluster using `ck8s-kubespray apply-ssh`. The prerequisites for doing this are:
+
+- A file containing the public SSH keys that should be authorized to access the cluster, separated by line breaks. Note that the authorized SSH keys for the cluster will be set to these keys _exclusively_, removing any keys that may already be authorized, so make sure the file includes **every SSH key** that should be authorized.
+- Set the environment variable `SSH_KEY_LOCATION` to the absolute path of the file.
```

Comment:
I'm not sure how to deal with this in a multi-VM setting, but for single VMs, we used to keep a "backup" SSH connection, just in case we cut ourselves off.

Is this still recommendable today? My thinking is that in worst case, you can always fix an SSH key misconfiguration by re-creating the VMs and putting the right keys in `cloud-init`.

Reply:
I'm not sure how this is done in the wild, but could it be an idea to have the script do it one-by-one and test the ssh connection between every change? If that's possible

Reply:
Certainly possible, but is it worth the effort? :smile:

Reply:
This should be possible with [serial](https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html#setting-the-batch-size-with-serial) or [throttle](https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html#restricting-execution-with-throttle) together with [step](https://docs.ansible.com/ansible/2.9/user_guide/playbooks_startnstep.html#step) or possibly a [confirmation step](https://stackoverflow.com/questions/34290525/pause-ansible-playbook-for-user-confirmation-whether-to-run-rest-tasks) I think.
Alternatively the [debug strategy](https://docs.ansible.com/ansible/latest/user_guide/playbooks_debugger.html#enabling-the-debugger-as-a-strategy) maybe?